pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix

```diff
@@ -214,4 +214,20 @@ in {
   # failing during the Setup.hs phase: https://github.com/gtk2hs/gtk2hs/issues/323.
   gtk2hs-buildtools = appendPatch ./patches/gtk2hs-buildtools-fix-ghc-9.4.x.patch super.gtk2hs-buildtools;
 
+  haskell-gi-base = __CabalEagerPkgConfigWorkaround super.haskell-gi-base;
```

Comment:
Still should be moved to be grouped with the other `__CabalEagerPkgConfigWorkaround` overrides above.